.flake8

```diff
@@ -1,37 +0,0 @@
-[flake8]
-#GOAL: max_line_length = 79 or 99
-max_line_length = 99
-per-file-ignores =
-    # too many leading '#' for block comment
-    */tests/*:E266
-    # line too long (several test strs)
-    test_granules.py:E501
-    # imported but unused
-    __init__.py:F401
-    # import not at top of file, imported but unused
-    doc/source/conf.py:E402,F401
-
-ignore =
-    # line too long
-    # NOTE: This is a formatting concern. Black handles long lines of code, but
-    # allows inline comments to be infinitely long (automatically formatting
-    # them can have unintended consequences). In our codebase, we have a lot of
-    # overlong comments.
-    # See: https://github.com/psf/black/issues/1713#issuecomment-1357045092
-    E501
-    # GOAL: remove ignores below this line
-    # comparison syntax in tests
-    E721
-    # bare except
-    E722
-    # ambiguous var name
-    E741
-    # unable to detect undefined names
-    F403
-    # line break before binary operator
-    # NOTE: This is a formatting concern
-    W503
-
-    # GOAL:
-    # syntax check doctests in docstrings
-    # doctests = True
```

Comment:
No way to syntax check docstrings in ruff yet - https://github.com/astral-sh/ruff/issues/3542. But we can at least format the docstrings, see below.

Reply:
Are we executing doctests? That'll give us the syntax check!

Reply:
No, not yet, but can do that later!